MIGRATIONS.md

```diff
@@ -3,6 +3,15 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 83
+@danielebogo 2018-10-30
+- Renamed `RevisionDiffAbstractValue`, `RevisionDiffContentValue`, `RevisionDiffTitleValue` to `DiffAbstractValue`, `DiffContentValue`, `DiffTitleValue`.
+- Set `DiffAbstractValue` as abstract entity.
+- Replaced relationship property name on `DiffContentValue` and `DiffTitleValue` from *relationship* to *revisionDiff*.
+- Fix parent entity of `DiffContentValue`.
+- Replaced properties name on `DiffAbstractValue`.
+- Added property *index* on `DiffAbstractValue`.
```

Comment:
I'm thinking this means that we added an `index` attribute to the `DiffAbstractValue` model but I don't see it in the editor.  Have I misunderstood?
<img width="849" alt="screen shot 2018-10-31 at 2 02 55 pm" src="https://user-images.githubusercontent.com/1435271/47812184-b7a69700-dd15-11e8-9dc6-97c3e7167340.png">

Reply:
I can see it on my model editor